api/app/fireweather_bot/hourly_actuals.py

```diff
@@ -42,12 +43,12 @@ def _get_end_date(self) -> datetime:
     async def run_wfwx(self):
         """ Entry point for running the bot """
         async with ClientSession() as session:
-            header = await wildfire_one.get_auth_header(session)
+            header = await wildfire_api.get_auth_header(session)
```

Comment:
I'd suggest that if we renamed `wildfire_api` to `wfwx_api`, it would be clearer that we're referring to an external API, not our own.

Reply:
Neither are clear to me that it's external is that what `wfwx` means?